pkgs/development/libraries/xxHash/default.nix

```diff
@@ -1,4 +1,10 @@
-{ lib, stdenv, fetchFromGitHub }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, fetchurl
+, enableShared ? !stdenv.hostPlatform.isStatic
```

Comment:
To answer your question in the other thread: we don't need this here. There are stdenv functions to achieve the same.